### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by RongXiefdu**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -121,6 +121,7 @@ export class SerialPort {
 }
 
 export class FileSystem {
```
Comment (dilin-MS) [inline src/index.ts:123]: Add comment above function using /** */. 
In this way, comment of function will appear when user hovers over function name.

File: src/index.ts [hunk 2]
```diff
@@ -149,6 +150,8 @@ export class FileSystem {
     )) as void;
   }
 
+  // remotePath: path of file to transfer in container
+  // localPath: path to save the transferred file on local machine
   static async transferFile(remotePath: string, localPath: string) {
```
Comment (dilin-MS) [inline src/index.ts:160]: /**
 *Transfer file from container to host machine.
 *@param remotePath: file path in contianer workspace.
 *@param localPath: file path in host machine.
 */

File: src/index.ts [hunk 3]
```diff
@@ -183,6 +186,8 @@ export class FileSystem {
     );
   }
 
+  // remotePath: path of folder to transfer in container;
+  // localPath: path to save the transferred folder on local machine
```
Comment (dilin-MS) [inline src/index.ts]: /**
*Transfer folder from container to host machine.
*@param remotePath: folder path in contianer workspace.
*@param localPath: folder path in host machine.
*/

File: src/index.ts (hunk 1 above)
Comment (dilin-MS) [reply src/index.ts:123]: image

File: src/index.ts [hunk 4]
```diff
@@ -216,6 +221,7 @@ export class FileSystem {
     );   
   }
 
+  // localPath: path to write the data to on local machine
```
Comment (dilin-MS) [inline src/index.ts]: /**
*In remote mode, write data to file in host machine.
*@param localPath: destination file path in host machine.
*@param data: data to write into file.
*/

File: src/index.ts [hunk 5]
```diff
@@ -367,6 +373,8 @@ export class SSH {
     )) as string;
   }
 
+  // localPath: path of file to upload on local machine
```
Comment (dilin-MS) [inline src/index.ts]: Same as above.

File: src/index.ts [hunk 6]
```diff
@@ -385,6 +393,8 @@ export class SSH {
     )) as void;
   }
 
+  // localFolderPath: path of folder to upload on local machine
```
Comment (dilin-MS) [inline src/index.ts]: Same as above.

File: src/index.ts [hunk 7]
```diff
@@ -367,6 +373,8 @@ export class SSH {
     )) as string;
   }
 
+  // localPath: path of file to upload on local machine
+  // remotePath: path to save the uploaded file on device such as Raspberry Pi
```
Comment (dilin-MS) [inline src/index.ts]: >device such as Raspberry Pi [](start = 51, length = 27)

target device. (Remove Raspberry Pi)

File: src/index.ts (hunk 1 above)
Comment (dilin-MS) [reply src/index.ts:123]: Type `/**` and enter, VS Code will scaffold the structure for you.

File: src/index.ts (hunk 7 above)
Comment (RongXiefdu) [reply src/index.ts]: Fixed.

File: src/index.ts (hunk 1 above)
Comment (RongXiefdu) [reply src/index.ts:123]: Fixed.

File: src/index.ts (hunk 3 above)
Comment (RongXiefdu) [reply src/index.ts]: Fixed.

File: src/index.ts (hunk 2 above)
Comment (RongXiefdu) [reply src/index.ts:160]: Fixed.

File: src/index.ts (hunk 4 above)
Comment (RongXiefdu) [reply src/index.ts]: Fixed.

File: src/index.ts (hunk 5 above)
Comment (RongXiefdu) [reply src/index.ts]: Fixed.

File: src/index.ts (hunk 6 above)
Comment (RongXiefdu) [reply src/index.ts]: Fixed.

**Review by dilin-MS — APPROVED** [review]

**Pull request merged**